Formula/mesa.rb

```diff
@@ -3,8 +3,8 @@ class Mesa < Formula
 
   desc "Graphics Library"
   homepage "https://www.mesa3d.org/"
-  url "https://archive.mesa3d.org/mesa-21.1.3.tar.xz"
-  sha256 "cbe221282670875ffd762247b6a2c95dcee91d0a34c29802c75ef761fc891e69"
+  url "https://mesa.freedesktop.org/archive/mesa-21.1.4.tar.xz"
+  sha256 "1f177f44098164b65731c5ded4c928fd58b14f6c9d2087aa0e37bc79bf79e90b"
```

Comment:
Why change the domain?

Reply:
It does redirect to the original domain:
```
❯ curl -Ls -o /dev/null -w %{url_effective} https://mesa.freedesktop.org/archive/mesa-21.1.4.tar.xz
https://archive.mesa3d.org//mesa-21.1.4.tar.xz
```
But we should stick with mesa3d.org anyway.